src/components/vacation-requests-table/vacation-requests-table-toolbar/vacation-requests-table-toolbar.tsx

```diff
@@ -72,7 +86,7 @@ const TableToolbar = ({
    * Delete vacation requests and statuses
    */
   const deleteVacationsData = async () => {
-    await deleteVacationRequests(selectedRowIds, rows);
+    await deleteVacationRequests(selectedRowIds, rows).then(() => setSelectedRowIds([]));
```

Comment:
Call `setSelectedRowIds` in the upper level component

Reply:
Moved call to an upper component